WordPressUtils/src/main/java/org/wordpress/android/util/helpers/MediaFile.java

```diff
@@ -381,4 +381,17 @@ public String getAttachmentPageURL(String siteUrl) {
         String attachmentPageUrl = UrlUtils.appendUrlParameter(siteUrl, "p", mMediaId);
         return attachmentPageUrl;
     }
+
+    public String getOptimalFileURL() {
+        String mediaURL;
+        if (!TextUtils.isEmpty(getFileUrlLargeSize())) {
+            mediaURL = getFileUrlLargeSize();
+        } else if (!TextUtils.isEmpty(getFileUrlMediumSize())) {
+            mediaURL = getFileUrlMediumSize();
+        } else {
+            mediaURL = getFileURL();
```

Comment:
NP: we can just reference the private members directly here (i.e. `mFileLargeURL`, etc.).

Reply:
Nice! I updated it in https://github.com/wordpress-mobile/WordPress-Utils-Android/pull/108/commits/2a392d1cadc26a67945205511019c26fd38bc788